cmd/capabilities_fixes.go

```diff
@@ -14,31 +14,32 @@ func fixCapabilitiesNIL(resource k8sRuntime.Object) k8sRuntime.Object {
 		if container.SecurityContext.Capabilities.Add == nil {
 			container.SecurityContext.Capabilities.Add = []Capability{}
 		}
-		containers = append(containers, container)
 	}
 	return setContainers(resource, containers)
 }
 
 func fixCapabilityNotDropped(resource k8sRuntime.Object, occurrence Occurrence) k8sRuntime.Object {
-	var containers []Container
-	for _, container := range getContainers(resource) {
-		container.SecurityContext.Capabilities.Drop = append(container.SecurityContext.Capabilities.Drop, Capability(occurrence.metadata["CapName"]))
-		containers = append(containers, container)
+	containers := getContainers(resource)
+	for _, container := range containers {
+		if occurrence.container == container.Name {
```

Comment:
Is container.name unique for every container inside a k8s resource?

Reply:
According to https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/, names are unique within namespaces but not necessarily across them.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L2089-L2093 So from what I take from the comment they should be unique.